tests/virt_autotest/guest_migration_dst.pm

```diff
@@ -14,7 +14,17 @@ use testapi;
 use lockapi;
 use mmapi;
 use upload_system_log 'upload_supportconfig_log';
-use virt_autotest::utils qw(is_xen_host);
+use virt_autotest::utils qw(is_xen_host is_kvm_host);
+use version_utils 'is_sle';
+
+#Since 15-SP4 kernel(v5.14), default value of unprivileged_userfaultfd sysctl is 0.
+#if unprivileged user want to use userfaultfd syscalls, we just only need to manually
+#reset unprivileged_userfaultfd value is 1 as the workaround on postcopy migration dst
+sub reset_unprivileged_userfaultfd {
+    script_run("sysctl -w vm.unprivileged_userfaultfd=1", 15);
+    script_run("sysctl -a | grep vm.unprivileged_userfaultfd", 15);
+    save_screenshot;
+}
```

Comment:
Could you help move this function to the bottom of this file guest_migration_dst.pm ?

Reply:
sure, had moved it now.